Signal/src/ViewControllers/MediaDetailViewController.m

```diff
@@ -725,19 +732,21 @@ - (void)presentFromViewController:(UIViewController *)viewController
                                    // Make sure imageView is layed out before we update it's frame in the next
                                    // animation.
                                    [self.mediaView.superview layoutIfNeeded];
+                                   [self applyFinalMediaViewConstraints];
 
                                    // 2. Animate imageView from it's initial position, which should match where it was
                                    // in the presenting view to it's final position, front and center in this view. This
-                                   // animation intentionally overlaps the previous
-                                   [UIView animateWithDuration:0.2
-                                       delay:0.08
+                                   // animation duration intentionally overlaps the previous
+                                   [UIView animateWithDuration:2.2
+                                       delay:2.08
```

Comment:
Was this just slowed down for debugging?  2 seconds delay/duration seems very long?

Reply:
Yikes, good catch. Thanks.